### PR TITLE
Adds Striped Normal-height Socks & All the Undergarments for Vox

### PR DESCRIPTION
**Pull request by KasparoVy**
**What does this PR do:**
Adds Vox-fitted versions of every undergarment in the game.
I think this is the last of the big-ticket items to sprite for Vox.

**Changelog:**
:cl:
add: Adds striped normal-height socks.
add: Vox can now wear whatever undergarment they want.
tweak: Removes flesh from shredded Vox uniform sprites.
del: Armalis can no longer wear undergarments. They didn't fit, anyhow.
/:cl:

**Images of sprite/map changes (IF APPLICABLE):**
_NOTE: If you're wondering why one of the Vox' legs looks wack it's 'cause it's prosthetic. I used my presaved character and didn't think otherwise until I was mostly done._
_[Previews also available here until Imgur drops it.](https://imgur.com/a/1gQbarA)_

_Undershirts_
para ss13 vox undershirts
_Underwear_
para ss13 vox underwear
_Socks_
para ss13 vox socks

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
img

oh no

**Comment by FreeStylaLT** [comment]
img
OH NO

**Comment by KasparoVy** [comment]
:sunglasses: Every single one

**Comment by Bookdrizzt** [comment]
Voxxy can has socks, voxxy is FREEEEEEE

File: code/modules/mob/living/carbon/human/species/vox.dm [hunk 1]
```diff
@@ -173,9 +173,9 @@
 	eyes = "blank_eyes"
 
 	species_traits = list(NO_SCAN, NO_BLOOD, NO_PAIN, IS_WHITELISTED)
-	dies_at_threshold = TRUE
+	clothing_flags = 0 //IDK if you've ever seen underwear on an Armalis, but it ain't pretty.
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/species/vox.dm]: Lol.

**Comment by Fox-McCloud** [comment]
Never again shall we have to suffer Armalis underwear.

File: code/modules/mob/living/carbon/human/species/vox.dm (hunk 1 above)
Comment (KasparoVy) [reply code/modules/mob/living/carbon/human/species/vox.dm]: Haha yeah I couldn't help but check and it was mega wack so I figured I'd fix it. I should've probably worded it like "seen an Armalis in their underwear" but oh well, hindsight 😂👍

**Comment by Kyep** [comment]
This makes the game unplayable for me. It prevents me (vulp char) from opening the char preferences window at all, with runtimes:
DEBUG: Runtime in preferences_setup.dm,387: Cannot read null.sprite_sheets [view]
DEBUG: Runtime in preferences.dm,237: bad resource file [view]

**Pull request merged**